torchdata/datapipes/utils/common.py

```diff
@@ -63,14 +64,14 @@ def _default_filepath_fn(data):
     return os.path.normpath(data)
 
 
-def validate_pathname_binary_tuple(data: Tuple[str, BufferedIOBase]):
+def validate_pathname_binary_tuple(data: Tuple[str, Union[BufferedIOBase, HTTPResponse]]):
```

Comment:
These two classes are sharing `io.IOBase`. Is there any concern that we have to use union of these two types?